WordPress/src/main/java/org/wordpress/android/ui/accounts/NewUserFragment.java

```diff
@@ -464,6 +487,26 @@ public void onClick(View v) {
     }
 
     @Override
+    public void onAttach(Context context) {
+        super.onAttach(context);
+
+        if (AppPrefs.isLoginWizardStyleActivated()) {
+            if (context instanceof LoginListener) {
+                mLoginListener = (LoginListener) context;
+            } else {
+                throw new RuntimeException(context.toString() + " must implement LoginListener");
+            }
+        }
+    }
+
+    @Override
+    public void onDetach() {
+        super.onDetach();
+
```

Comment:
Handled in a different PR (https://github.com/wordpress-mobile/WordPress-Android/pull/6164#discussion_r126885786) so, I'll leave as is in this one.